.github/BOTMETA.yml

```diff
@@ -260,7 +260,7 @@ files:
   $modules/monitoring/monit.py: brian-brazil
   $modules/monitoring/pagerduty.py:
     ignored: bpennypacker
-  $modules/monitoring/zabbix/: eikef D3DeFi rubentsirunyan
+  $modules/monitoring/zabbix/: eikef D3DeFi rubentsirunyan sky-joker
```

Comment:
@D3DeFi Thanks for the PR. Could you please create `team_zabbix` and add all names from here just like `team_ipa` or `team_vmware` ?

Reply:
Thanks, changed it to team_zabbix!